lib/src/values/logic_value.dart

```diff
@@ -7,6 +7,7 @@
 /// 2021 August 2
 /// Author: Max Korbel <max.korbel@intel.com>
 ///
+// ignore_for_file: prefer_single_quotes
```

Comment:
Let's not add this lint ignore, especially for the whole file.

If you're trying to split a line across multiple lines, you can do 
```dart
'''multi-line
string
'''
```
(i.e. it doesn't have to be double-quotes)

Also, you can just split strings like this, if you're just trying to make it fit on a line:
```dart
'this string needs'
' to be split across multiple lines'
' even though it is not a mult-line string'
```

You don't even need any `+`'s between them